spec/controllers/companies_controller_spec.rb

```diff
@@ -0,0 +1,151 @@
+require 'rails_helper'
+
+RSpec.describe CompaniesController, type: :controller do
+  let(:user) { create :user }
```

Comment:
always use "let!" instead of "let"

Reply:
changed it, but in this case it was 'let' cause user was only used in 5th line for signing_in before each test